projects/cashmere/src/lib/button/split-button/split-button.component.ts

```diff
@@ -18,13 +20,22 @@ export class SplitButtonComponent {
     private _disabled: boolean = false;
     private _color: string = 'primary';
 
+    /* primary button's click event */
     @Output() click = new EventEmitter<SplitButtonClickEvent>();
 
+    /* Additional information shown as tooltip */
     @Input() title: string;
+
+    /* Type of button */
     @Input() type = 'button';
+
+    /* Used as a reference in JavaScript, or to reference form data after a form is submitted */
     @Input() name: string;
+
+    /* Value of button */
```

Comment:
Might be a little clearer to explain that this is the text that will appear on the button.